modules/rosa-sts-account-roles-terraform.adoc

```diff
@@ -0,0 +1,193 @@
+  operator_role_policies = data.rhcs_policies.all_policies.operator_role_policies
+  all_versions           = data.rhcs_versions.all
+  tags                   = var.tags
+}
+----
+
+.. Copy this template into a file  named `output.tf`. In the `output.tf` file, the account role prefix structure is defined. This output definition allows you to specify how the various generated roles are constructed. 
++
+[source,terraform]
+----
+output "account_role_prefix" {
+  value = module.create_account_roles.account_role_prefix
+}
+----
+
+.. Copy this template into a file  named `variables.tf`. You can provide your own variables in the default fields for the account-role creation variables. 
++
+[IMPORTANT]
+====
+Do not include your OCM token in this file if it is not stored in a safe location.
+====
++
+[source,terraform]
+----
+variable "openshift_version" {
+  type = string
+  default = "4.13"
+  description = "Enter the desired OpenShift version as X.Y. This version should match what you intend for your ROSA cluster. For example, if you plan to create a ROSA cluster using `4.13.10`, then this version should be `4.13`. You can see the supported versions of OpenShift by running `rosa list version`."
+}
+
+variable "account_role_prefix" {
+  type    = string
+  default = ""
+  description = "Your account roles are prepended with whatever value you enter here. The default value in the ROSA CLI is `ManagedOpenshift-` before all of your account roles."
+}
+
+variable "tags" { <1>
+  description = "(Optional) List of AWS resource tags to apply."
+  type        = map
+  default     = null
+}
+----
++
+--
+<1> The `tags` parameter uses a map of strings variable type. The format that it takes looks like the following example:
++
+[source,terminal]
+----
+variable "tags" {
+  type    = "map"
+  default = {
+    "us-east-1" = "image-1234"
+    "us-west-2" = "image-4567"
+  }
+}
+----
+--
+. In your terminal in the directory where you saved these Terraform files, run the following command to set up Terraform to create these resources:
++
+[source,terminal]
+----
+$ terraform init
+----
+. You can optionally run the following command to confirm that the Terraform you copied is correct:
++
+[source,terminal]
+----
+$ terraform validate
+----
++
+.Sample output
++
+[source,terminal]
+----
+Success! The configuration is valid.
+----
+. You can test your Terraform template and create a reusable Terraform plan file by running the following command:
++
+[source,terminal]
+----
+$ terraform plan -out account-roles.tfplan -var token=<OCM_token>
+----
+. Run the following command to build your account-wide IAM roles with Terraform:
++
+[source,terminal]
+----
+$ terraform apply "account-roles.tfplan"
+----
++
+[NOTE]
+====
+If you used the `terraform plan` command first, you can provide your created account-roles.tf file here. If you do not, Terraform temporarily creates this plan before it applies your desired outcome.
```

Comment:
File name should be in backticks (plus a word choice suggestion, feel free to ignore this part):

```suggestion
If you used the `terraform plan` command first, you can provide your created `account-roles.tf` file here. Otherwise, Terraform temporarily creates this plan before it applies your desired outcome.
```